### PR TITLE
feat(tracing): add http.client_ip attribute

### PR DESCRIPTION
**Pull request by backjo**
### Summary

As a developer troubleshooting requests using trace data, I expect my access logs to be consistent with my traces. The access logs currently record the forwarded_ip, which tells me what my client's true ip is (assuming proper setup of trusted IPs etc). This change adds a span attribute for client ip using the forwarded ip
### Checklist

- [x] The Pull Request has tests
- [x] There's an entry in the CHANGELOG
- [ ] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - PUT DOCS PR HERE

### Full changelog

* [Implement ...]

### Issue reference

Fix #_[issue number]_

### CODE REVIEW

File: kong/tracing/instrumentation.lua [hunk 1]
```diff
@@ -236,7 +236,7 @@ function _M.request(ctx)
       ["http.host"] = host,
       ["http.scheme"] = scheme,
       ["http.flavor"] = http_flavor,
-      ["net.peer.ip"] = client.get_ip(),
+      ["net.peer.ip"] = client.get_forwarded_ip(),
```
Comment (attenuation) [inline kong/tracing/instrumentation.lua:239]: I think Kong using `["net.peer.ip"] = client.get_ip(),` is Okay for the Kong use case, if you want to add `forward` IP attribute to tracing instrumentation PDK, Maybe you can add a new entry like `["net.peer.forwarded_ip"]`, but I am not sure if this is Okay for tracing system.

Comment (backjo) [reply kong/tracing/instrumentation.lua]: updated and pushed

**Comment by attenuation** [comment]
@backjo Could you help us to fix the test? This is related to your feature.

**Comment by backjo** [comment]
@attenuation will update this PR after the earlier 

> @backjo Could you help us to fix the test? This is related to your feature. https://github.com/Kong/kong/blob/master/spec/02-integration/14-tracing/01-instrumentations_spec.lua#L138

Should be fixed - added test attribute to verify this new one

**Comment by StarlightIbuki** [comment]
Thanks for your contribution. Please add a proper changelog.

**Review by attenuation — COMMENTED** [review]
From https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#http-client, 

> The IP address of the original client behind all proxies, if known (e.g. from [X-Forwarded-For](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For)). [2]

That means: 
`http.client_ip` is the real client IP
`net.peer.ip` is the proxy socket source IP.

**Comment by backjo** [comment]
Added the changelog!

**Review by hanshuebner — APPROVED** [review]

**Pull request merged**